concepticondata/concepticon.tsv

```diff
@@ -3181,4 +3181,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3186	BE DEAD OR DEATH OR DIE	The body	To be not alive or the end of a life or to cease to live.	Other	
 3187	DROWNED	The body	To have perished by suffocation in water.	Property	
 3188	MIGHTY OR POWERFUL OR STRONG		Holding much (political, social, ...) power or being done using a huge amount of energy or having great physical strength.	Property	
-
+3189	STAB TO DEATH	The body	To kill by stabbing with a knife.	Action/Process	
+3190	BROKEN OR SHATTERED	Basic actions and technology	Not capable of working or broken into pieces.	Property	
```

Comment:
I don't really understand this definition. I also propose to replace the glosses for broken and shattered by "broken (into pieces)" and "broken". Shattered is not the nromal word one uses often, so the gloss is confusing.

Reply:
I wasn't sure about calling it SHATTERED. But some entries on "BROKEN" refer to "broken into pieces", some to "not working", one clearly to both, the rest are ambiguous and the definition is "not working", so I decided to rather move the "broken into pieces" concepts to a new concept. We could instead also change the current definition of "BROKEN" to "broken into pieces" and instead add a new concept "NOT WORKING" for that meaning.